include/inputstr.h

```diff
@@ -374,8 +374,8 @@ typedef struct _ButtonClassRec {
 } ButtonClassRec, *ButtonClassPtr;
 
 typedef struct _FocusClassRec {
```

Comment:
Right now this struct is in the public ABI - we would have to make it private first (next major release).
Feel free to submit a PR (against rfc/new-abi-25.1) for this.

I don't think that nvidia's using it, so we could do it with next major (when input drivers need to be recompiled anyways).